tests/integration/helpers.py

```diff
@@ -123,7 +125,7 @@ async def get_alertmanager_alerts(ops_test: OpsTest, unit_name, unit_num, retrie
     """
     # TODO consume this from alertmanager_client when becomes available
     address = await get_unit_address(ops_test, unit_name, unit_num)
-    url = f"http://{address}:9093/api/v2/alerts"
+    url = f"http://{address}:9093{path}/api/v2/alerts"
```

Comment:
This feels very brittle. We should be asserting that path begins with / and does not end with /
alternatively we can (ab)use pathlib.Path to merge the two, and do something like
```
url = f"http://{address}:9093/" + str(Path(path.strip("/") / "api" / "v2" / "alerts")
```

Reply:
`)` missing

```suggestion
    url = f"http://{address}:9093/" + str(Path(path.strip("/")) / "api" / "v2" / "alerts")
```

Reply:
Iirc, pathlib and urlparse treat "path" differently with respect to leading/trailing slashes.

I think that for `path=""` we'd need the following to avoid getting `//`:
```python
path = "/" + path.lstrip("/").rstrip("/")
url = f"http://{address}:9093{path}/api/v2/alerts/groups"
```